kamu-adapter-graphql/src/scalars/dataset.rs

```diff
@@ -178,6 +178,14 @@ pub struct DataBatch {
 }
 
 impl DataBatch {
+    pub fn empty(format: DataBatchFormat) -> DataBatch {
+        DataBatch {
+            format,
+            content: String::from("{}"),
```

Comment:
This is incorrect since `format` can be many things other than `JSON` and `{}` will be invalid for CSV, JsonLD and other supported formats. See `from_records` below as I had to add some code to handle this already.

Reply:
Changed string value depending on the format.  assume `JsonXXX` are still `Json`